volatility3/framework/plugins/linux/sockstat.py

```diff
@@ -591,7 +593,7 @@ def _generator(self, pids: List[int], netns_id_arg: int, symbol_table: str):
             tasks: String with a list of tasks and FDs using a socket. It can also have
                    extended information such as socket filters, bpf info, etc.
         """
-        filter_func = lsof.pslist.PsList.create_pid_filter(pids)
```

Comment:
Yep, this was bad and I think I prefer the direct requirements (even though it would technically come bundled with lsof).  Good spot!